src/components/AssetManagementPanel/AssetManagementApplication/index.tsx

```diff
@@ -46,11 +46,27 @@ export const AssetManagementApplication: FunctionComponent<AssetManagementApplic
     restoreTokenState,
   } = useTokenInformationContext();
   const [assetManagementAction, setAssetManagementAction] = useState(AssetManagementActions.None);
-  const { upgradeToMetaMaskSigner, account, provider } = useProviderContext();
+  const [account, setAccount] = useState<string | undefined>();
+  const { upgradeToMetaMaskSigner, getSigner, getProvider } = useProviderContext();
+
+  const provider = getProvider();
   const { tokenRegistry } = useTokenRegistryContract(tokenRegistryAddress, provider);
   // Check if direct owner is minter, useContractFunctionHook value returns {0: boolean}
   const { call: checkIsMinter, value: isMinter } = useContractFunctionHook(tokenRegistry, "isMinter");
 
+  useEffect(() => {
+    const updateAccount = async () => {
+      try {
+        const signer = getSigner();
+        const address = signer ? await signer.getAddress() : undefined;
+        setAccount(address);
+      } catch (_) {
+        setAccount(undefined);
+      }
```

Comment:
here also?

Reply:
Ahh... Updated in https://github.com/TradeTrust/tradetrust-website/pull/506/commits/e7d904f5c39cab8d2852511826ae6ba6139e2f34